.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,246 @@
+name: Build and upload to PyPI
+
+on:
+  pull_request:
+    branches:
```

Comment:
Will this trigger only on branches that start with package- ?

Could you add docs that describe the release workflow to make clearer how it's intended to work?

Reply:
I've updated the wiki notes here: https://github.com/MDAnalysis/mdanalysis/wiki/PreparingReleases

We keep a copy of this in the userguide which I'll update once this PR is merged.

Reply:
Thank you — I agree with keeping things DRY and to eventually have docs in only one place.